modules/analytics/analytics.js

```diff
@@ -123,6 +123,13 @@ class Analytics {
                     permanentProperties.group = group;
                 }
 
+                if (window.jitsiRegionInfo) {
```

Comment:
@saghul thanks for letting us know that such agreement exists. I will propose on next retro that we add such agreements to our coding style .md file into "special agreements" section or something.